source/docs/robot-design/odometry.rst

```diff
@@ -39,53 +115,58 @@ encoders. If you're using dead wheels, you will not need the drive
 motor encoder ports, so those are potential ports you might want
 to use.
 
-There are really only two good options for odometry,
-specifically deadwheels. Otherwise you will want to just
-use the encoders that come with the motors you are using
-for the drivetrain if instead you would rather utilize
-the drive encoders (which is not recommended).
-
 REV Through-Bore
------------------
+----------------
 Often short-handed to "REV-coders" or "revcoders,"
 the `REV Through-Bore encoders <https://www.revrobotics.com/rev-11-1271/>`_
-is the option that has gained significant traction in
-the FTC community for odometry is because of how
-easy it is to set up and design with. Unlike
-previously used encoders like the E8T, the through-bore
-does not have a sliding disc.
-
-Some reasons teams might want to consider using the
-revcoders for their odometry are:
-
-* It's magnetic and won't break easily
-* Cheap
-* Large CPR
-* Easy wiring
-
-However, there will be velocity overflow if not handled properly.
-Due to the high tick count, the velocity of the
-encoder can result in integer overflow due to the REV Hub bus
-using 16-bit signed values for sensor data.
+has quickly become the de facto option the FTC community. The REV
+encoders have gained such a reputation due to its relative affordability,
+much improved reliability, and ease of use. The through-bore design
+proves to be a *significant* improvement over previous optical disc
+encoder designs. Optical disc encoders are very fragile, prone to
+scratching, and are much less tolerant to design flaws.
+
+**Advantages:**
+
+    * Through-bore design is very robust
+    * Relatively cheap
+    * High CPR
+    * Easy wiring
+
+**Disadvantages:**
+
+    * Quite large relative to other encoders. May be challenging to
+      create a compact design
+
+**Note:** The Through-Bore encoders have a very high CPR (8k). The Rev
+Hub stores velocity in a 16-bit signed integer. This means it can store a
+maximum value of 2^15 (which is 32768). Thus, it only takes 4 rotations
+a second (32k / 8k = 4) for the velocity value on the Rev Hub to
+experience an
+`integer overflow <https://en.wikipedia.org/wiki/Integer_overflow?oldformat=true>`_.
+This is only a concern when dealing with motion profiling. The existing tools
+(Road Runner and FTCLib) have mechanisms for deadling with this issue.
```

Comment:
```suggestion
.. note::
   The Through-Bore encoders have a very high CPR (8k). The REV Hub stores
   velocity in a 16-bit signed integer. This means it can store a maximum value
   of 2^15 (which is 32768) ticks per second. Thus, it only takes 4 rotations a
   second (32k / 8k = 4) for the velocity value on the REV Hub to experience an
   `integer overflow
   <https://en.wikipedia.org/wiki/Integer_overflow?oldformat=true>`_. This is
   only a concern when dealing with motion profiling. The existing localization
   software (Road Runner and FTCLib) have mechanisms for deadling with this
   issue.
```